src/shared/components/palette/base-field/field-read.component.ts

```diff
@@ -28,6 +28,10 @@ export class FieldReadComponent extends AbstractFieldReadComponent implements On
   }
 
   ngOnInit(): void {
+    // Ensure this.caseField is actually a CaseField instance even if instantiated with {}
+    if (!(this.caseField instanceof CaseField)) {
+      this.caseField = plainToClassFromExist(new CaseField(), this.caseField);
```

Comment:
👍 
would the write component have the same issue?

Reply:
added to PR description...